tmt/options.py

```diff
@@ -167,6 +168,37 @@
     ]
 
 
+LINT_OPTIONS: List[ClickOptionDecoratorType] = [
+    click.option(
+        '--list-checks',
+        is_flag=True,
+        help='List all available checks.'),
+    click.option(
+        '--enable-checks',
+        metavar='CHECK-ID',
+        multiple=True,
+        type=str,
+        help='Run only checks mentioned by this option.'),
+    click.option(
+        '--enforce-checks',
+        metavar='CHECK-ID',
+        multiple=True,
+        type=str,
+        help='Consider linting as failed if any of the checks is not a pass.'),
+    click.option(
+        '--failed-only',
+        is_flag=True,
+        help='Display only tests/plans/stories that fail a check.'),
+    click.option(
+        '--outcome-only',
+        metavar='OUTCOME',
+        multiple=True,
+        type=click.Choice([
+            member.value for member in tmt.lint.LinterOutcome.__members__.values()]),
+        help='Display only checks with the given outcome.')
```

Comment:
Could we  show the list of possible OUTCOME values? Currently one has to dig it from error message printed after invalid input value.

Reply:
Done in 69a3c6f1:

```
  --outcome-only skip|pass|fail|warn|fixed
                                  Display only checks with the given outcome.
```